dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -239,3 +239,7 @@ scheme.incorrect.error.useraction=The scheme type must be http, https, ws, or ws
 upgradeHttpToWebSocket.failed=CWWKH0055E: The jakarta.websocket.server.ServerContainer#upgradeHttpToWebSocket call failed due to {0}. 
 upgradeHttpToWebSocket.failed.explanation=The server could not upgrade the HTTP or HTTPS request to the ws/wss protocol.
 upgradeHttpToWebSocket.failed.useraction=Review the error to identify why the request failed to upgrade.
+
+maxmessagesize.exceeded=CWWKH0056E: The maxMessageSize for method {0} exceeded Integer.MAX_VALUE for class {1}
```

Comment:
I would add a period to these three messages.

Maybe also add "as per the WebSocket 2.2 spec".